### PR TITLE
implement profile component

### PR DESCRIPTION
**Pull request by maya-abdoh**

### CODE REVIEW

File: UserManagementTeam/src/Profile.vue [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<template>
+    <div class="profile-container">
+      <div class="profile-card">
+        <div class="profile-content">
+          <div class="profile-info">
+            <p><strong>User Name:</strong> {{ profile.name }}</p>
+            <p><strong>Email:</strong> {{ profile.email }}</p>
+            <p><strong>Phone:</strong> {{ profile.phone }}</p>
```
Comment (noorhonjol) [inline UserManagementTeam/src/Profile.vue]: the field you render not compatible with what in interface

**Review by noorhonjol — CHANGES_REQUESTED** [review]

File: UserManagementTeam/src/index.css [hunk 2]
```diff
@@ -1,10 +1,11 @@
...
-  font-size: 3rem;
-  margin: auto;
-  max-width: 800px;
-  margin-top: 20px;
+.custom-card-shadow {
+  box-shadow: 0 8px 16px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+}
+.custom-label-border {
+  border-image-slice: 1;
+  border-image-source: linear-gradient(135deg, #71b7e6, #9b59b6);
```
Comment (noorhonjol) [inline UserManagementTeam/src/index.css]: there is shadow in tailwind and daisyUI you dont need to write css

File: UserManagementTeam/src/Profile.vue [hunk 3]
```diff
@@ -1,23 +1,39 @@
...
-      <div class="profile-card">
-        <div class="profile-content">
-          <div class="profile-info">
-            <p><strong>User Name:</strong> {{ profile.name }}</p>
-            <p><strong>Email:</strong> {{ profile.email }}</p>
-            <p><strong>Phone:</strong> {{ profile.phone }}</p>
-          </div>
+  <div class="profile-container flex justify-center items-center min-h-screen">
+    <div class="profile-card card w-full max-w-md h-[600px] bg-base-100 custom-card-shadow rounded-3xl p-8">
+      <div class="profile-content card-body mt-10">
```
Comment (noorhonjol) [inline UserManagementTeam/src/Profile.vue]: profile-container  ,profile-card , profile-content , profile-info not useful you can delete it

**Review by noorhonjol — CHANGES_REQUESTED** [review]

File: UserManagementTeam/src/Profile.vue (hunk 3 above)
Comment (maya-abdoh) [reply UserManagementTeam/src/Profile.vue]: done

File: UserManagementTeam/src/index.css (hunk 2 above)
Comment (maya-abdoh) [reply UserManagementTeam/src/index.css]: done

**Review by noorhonjol — APPROVED** [review]

**Pull request merged**